src/components/WorldListPage/WorldListPage.tsx

```diff
@@ -53,28 +53,33 @@ const WorldListPage: React.FC<Props> = props => {
     worldsWalletStats,
     onNavigate,
     onOpenYourStorageModal,
-    onOpenWorldsForENSOwnersAnnouncementModal
+    onOpenWorldsForENSOwnersAnnouncementModal,
+    onUnpublishWorld
   } = props
   const [sortBy, setSortBy] = useState(SortBy.ASC)
   const [page, setPage] = useState(1)
   const { tab } = useCurrentlySelectedTab()
 
-  const isWorldDeployed = (ens: ENS) => {
-    if (ens.worldStatus?.healthy) {
-      const deployment = deploymentsByWorlds[ens.subdomain]
-
-      return deployment && deployment.projectId && !!projects.find(project => project.id === deployment.projectId)
-    }
+  const isWorldDeployed = useCallback(
+    (ens: ENS) => {
+      if (ens.worldStatus?.healthy) {
+        return !!deploymentsByWorlds[ens.subdomain]
```

Comment:
The previous condition validated that a world is published only when the scene project exists but didn't consider scenes deployed using sdk-cli or if the scene project has been deleted.